docs/components/PkgLayout.tsx

```diff
@@ -1,11 +1,10 @@
 import { PropsWithChildren } from 'react';
 import { useRouter } from 'next/router';
-import { CallToActionLink } from '@ag.ds-next/call-to-action';
-import { Prose } from '@ag.ds-next/prose';
-import { SkipLinksProps } from '@ag.ds-next/skip-link';
-import { SubNav } from '@ag.ds-next/sub-nav';
-import { Text } from '@ag.ds-next/text';
-import { PageAlert } from '@ag.ds-next/page-alert';
+import { CallToActionLink } from '@ag.ds-next/react/call-to-action';
+import { SkipLinksProps } from '@ag.ds-next/react/skip-link';
+import { SubNav } from '@ag.ds-next/react/sub-nav';
+// import { Text } from '@ag.ds-next/react/text';
```

Comment:
This has been commented out

Reply:
I've left this commented out as I would like to keep the functionality of displaying the warning "This package is yet to be released on npm." to users, but I'm not sure how we do this using the current single package architecture. I've just added a TODO comment to this file which captures this.